atd-vzd/triggers/update_noncr3_location.sql

```diff
@@ -0,0 +1,51 @@
+-- Function for insert and update triggers on atd_apd_blueform
+CREATE OR REPLACE FUNCTION update_noncr3_location()
+  RETURNS TRIGGER 
+  LANGUAGE PLPGSQL
+  AS
+$$
+DECLARE 
+    found_location_id varchar;
+
+BEGIN
+    -- Check if crash is on a major road and of concern to TxDOT.
+    -- NEW.position is recalculated in BEFORE trigger to ensure it is up to date
```

Comment:
i'm not familiar with the distinction between `position` and the lat/lon columns, but fwiw triggers on the same event fire in alphabetical order, in case you did want to chain them together ([docs](https://www.postgresql.org/docs/9.5/trigger-definition.html)):

> If more than one trigger is defined for the same event on the same relation, the triggers will be fired in alphabetical order by trigger name

Reply:
TIL - I think that could be a better option here so there isn't the extra dependency of order of `AFTER` triggers. I'll make this change. 🙏